onnxruntime/test/util/include/tensors_from_file.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <unordered_map>
+
+namespace onnxruntime {
+namespace test {
+
+// Read float tensors from a file
+void load_tensors_from_file(const std::string& path, std::unordered_map<std::string, std::vector<float>>& tensors);
```

Comment:
Coding style: functions should start with a capital letter and have a capital letter for each new word.

Reply:
You may also store them on disk as protobuf objects. Protobuf has a text format, which can save the need of writing parsing code.

Reply:
Fixed coding style.

I did not choose protobuf objects since I do not want to add dozens of files. I agree that it is a good option.

Reply:
One binary file could contain multiple pb objects, if you do not mind using a binary format.